2.adoc

```diff
@@ -72,5 +72,165 @@ Optional description
+
+==== Porovnání rozdílů
+Jedná sa o desktopový editor, nie webovú komponentu. 
+Podporuje iba jazyk markdown (github-flavored). 
+Kvôli chýbajúcej command pallete vyžaduje používanie myši ak si užívateľ nepamätá klávesovú skratku. 
+Podporuje zmenu vzhľadu pomocou css šablón.
+
+
+==== Zhodnocení UI
+
+Defaultné rozhranie so zobrazenou outline dokumentu:
+
+image::https://i.imgur.com/n3vyN3i.png[]
+
+* Focus mode
+** Celý dokument vybledne, zvýraznený je len aktívny blok
+
+image::https://i.imgur.com/u6NGu0Z.png[]
+
+* typewriter mode
+** Zaručí aby aktívny riadok bol vždy v strede editoru. 
+
+* Source code mode
+  ** Namiesto priamo live renderingu je stále zobrazený zdrojový kód, ktorý je zvýraznený nie len farebne ale aj veľkosťou písma, odsadením (vlastne vyrenderovaný dokument kde je miesto textu stále markdown).  
+  ** Pri zapnutí tohto módu nefunguje drag n drop na pridanie obrázku do dokumentu
+
+image::https://i.imgur.com/NWJ2Mtz.png[]
+
+* Outline 
+  ** podporu zbalenia/rozbalenia je nutné zapnúť v nastaveniach
+  ** UX fail: linky len na texte, nie na celom riadku outline.
+  ** Pevná šírka 
+* Podpora viacerých tabov (UX fail: ale každý dokument ako aj nápovedu vyvolanú z menu otvorí v novom okne)
+* UX fail: defaultne neexistuje klávesová skratka na otvorenie nového tabu s dokumentom (štandardne cmd+t)
+* Chýbajú presety klavesových skratiek 
+* Nápoveda je formou markdown dokumentu 
+
+==== Pozorování UI v kontextu markup-editor
+* Chýba command paleta
+  ** Alternatívou ku klávesovým skratkám je klasické menu, kde je užívateľ nutený siahnúť po myši
+  ** klávesové skratky na styling dokumentu (nadpisy, odsadenie, tabulky, bloky kódu, citacie, zoznamy, foot notes, horizontal lines, bold, underline,..., vloženie obrázku) (neaktívne v source code mode)
+* Nepodporuje režim so zobrazeným zdrojákom a vyrenderovaným dokumentom vedľa seba
+  ** štandardne live rendering v tom istom okne (preview)
+  ** source code mod pridáva nadštandardný syntax highlighting zdrojaku (niečo ako vyrenderovaný dokument akurát je v ňom stále markdown)
+* Chýba statusbar
+  ** Chýba informácia o pozícií kurzora v dokumente
+  ** Auto-save na macOS v réžií systému, bez informácie o poslednom uložení
+* Outline 
+  ** chýba presúvanie kapitol potiahnutím nadpisu v outline
+* Práca sa tabuľkami
+  ** UI dialog pre vloženie tabuľky (počet riadkov, sĺpcov)
+  ** Tabuľka je hneď vyrenderovaná a do buniek je možné vpisovať hodnoty
+  ** Malý toolbar pre výber zarovnania textu v bunkách tabuľky a pohodlný resize
+
+image::https://i.imgur.com/1uSz1xM.png[]
+
+* Vkladanie obrázkov
+  ** cez menu (otvorí sa nativny dialog), klávesovou skratkou (snippet) alebo drag&drop
+  ** možnosť automatického uploadu obrázku na server (nutné nainštalovať ďalšiu aplikáciu)
+
+#### Zajímavé features
+
+(viac menej asi popísané vyššie)
+
+* Pekný syntax highlighting kódu
```

Comment:
Jako chovanim nebo vizualne? Pokud jen vizualne tak to neni uplne zalezitost UI, leda by to melo nejaky pozorovatelny, pojmenovatelny vlastnosti, napriklad, ze to pouziva proporcionalne velky pisma nadpisu, nebo hodne paddingu oproti textaku a tak to pusobi prehledneji.